test/lib/dialogs/inst_product_license_test.rb

```diff
@@ -54,4 +54,12 @@
       dialog.contents
     end
   end
+
+  describe "#abort_handler" do
+    it "returns true if user confirm abort" do
```

Comment:
np: would be nice to cover both cases, so it returns whether the user confirmed to abort or not....